backend/slackbot/worker.js

```diff
@@ -159,8 +170,14 @@ async function handleScheduledReport() {
 
   // Post all of the reports to slack
   for (let message in slackResponse) {
-    await axios.post(slackURL, slackResponse[message], { headers: headers });
+    try {
```

Comment:
Above on line 37, you have a try/catch around a call to `handleScheduledReport` - because you're handling the error here, that error will never make it up to the calling function. You have two options: remove the try/catch on line 37, or `throw` the error in the catch block here.